app/models/claim.rb

```diff
@@ -246,6 +246,10 @@ def school
     eligibility&.current_school
   end
 
+  def days_since_submission
+    ((Time.now - submitted_at) / 1.day).to_i if submitted?
```

Comment:
I'm not entirely happy about this line, my gut says there should be a cleaner way to do it.

Reply:
See above, you probably don't need this. For the record though, the best way to get the number of days since a particular date would be this:

```ruby
(Date.today.to_date - submitted_at.to_date).to_i
```